diff --git a/dist/index.cjs b/dist/index.cjs
index aad651e68bd5cacc263a6a8027d285a4d6602af9..3dacb729655ff6b9b6fd1fad27a1602abd43e072 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -2328,7 +2328,7 @@ async function transformMain(code, filename, options, pluginContext, ssr, asCust
   if (devToolsEnabled || devServer && !isProduction) {
     attachedProps.push([
       `__file`,
-      JSON.stringify(isProduction ? path.basename(filename) : filename)
+      JSON.stringify(path.basename(filename))
     ]);
   }
   if (devServer && devServer.config.server.hmr !== false && !ssr && !isProduction) {
diff --git a/dist/index.mjs b/dist/index.mjs
index 59b670095d82c66f7eb904cb71dea9c2a079224e..afb29b29270b7b03dd716790719709833c15191a 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -2324,7 +2324,7 @@ async function transformMain(code, filename, options, pluginContext, ssr, asCust
   if (devToolsEnabled || devServer && !isProduction) {
     attachedProps.push([
       `__file`,
-      JSON.stringify(isProduction ? path.basename(filename) : filename)
+      JSON.stringify(path.basename(filename))
     ]);
   }
   if (devServer && devServer.config.server.hmr !== false && !ssr && !isProduction) {